src/node/index.ts

```diff
@@ -22,7 +22,7 @@ interface Service {
 interface BeaconNodeCtx {
   chain?: object;
   db?: object;
-  eth1?: object;
+  eth1?: any;
```

Comment:
Can we use an alternative to `any`?

Reply:
Probably should have put interface there. I'm not sure why objects are type of each config haha
Will try to put Eth1Options interface and check if it breaks anything

Reply:
Nope, doesn't work, since Eth1Notifier requires all props, and all of beacon opts should be optional so it has to be any :/

Reply:
ahh too bad ... no worries

Reply:
You've got my approval